scripts/ecr-public-login

```diff
@@ -16,4 +16,4 @@ function exit_and_fail() {
 
 trap exit_and_fail INT TERM ERR
 
-docker login --username AWS --password="$(aws ecr-public get-login-password --region us-east-1)" "${ECR_REGISTRY}"
\ No newline at end of file
+aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin "${ECR_REGISTRY}"
```

Comment:
is there any reason for modifying this command? Usually we don't prefer to change these login commands.

Reply:
Not really. I'll keep it as is!